typing/env.ml

```diff
@@ -104,6 +104,10 @@ let add_label_usage lu usage =
     lu.lu_mutation <- true;
     lu.lu_construct <- true
 
+let is_mutable_label_usage = function
+  | Mutation -> true
+  | (Projection | Construct | Exported_private | Exported) -> false
```

Comment:
I find the name of this function a bit misleading, as it is not about "mutable" labels but about "mutating" labels. One could also inline the function directly in the use site.

Reply:
(Amusingly, I actually asked Chris to outline this function in the first place)